bugsnag-android-core/src/main/java/com/bugsnag/android/Exceptions.java

```diff
@@ -72,7 +72,7 @@ private void exceptionToStream(@NonNull JsonStream writer,
         writer.name("message").value(message);
         writer.name("type").value(exceptionType);
 
-        Stacktrace stacktrace = new Stacktrace(frames, projectPackages);
+        Stacktrace stacktrace = new Stacktrace(frames, projectPackages, NoopLogger.INSTANCE);
```

Comment:
why does a stacktrace which is created here get a noop logger, but a stacktrace for a thread gets an actual logger?

Reply:
The `Exceptions` class will be deleted in #673 - so I've just passed in a `NoopLogger` here as its the path of least resistance.